address-space-controller/src/main/resources/templates/brokered-space-infra.yaml

```diff
@@ -193,6 +211,9 @@ objects:
         - name: external-cert
           secret:
             secretName: ${MESSAGING_SECRET}
+        - name: broker-support
```

Comment:
Wondering if we should mark this as optional?   We always expect the secret to be present, but breaking in the case where it is absent seems excessive.  WDYT?

Reply:
We could also take it a step further, to have the admin configure the secret with the credentials and point to it in the infra config for instance. I think it depends on how we think this feature would be used. For now, I'm thinking having it required is fine since its always set by the address-space-controller.